gitea/attachment.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package gitea // import "code.gitea.io/sdk/gitea"
+import "time"
+
+// a generic attachment
+type Attachment struct {
```

Comment:
At least in SDK I think we should call it also as `Asset` so that there would not be different names for same thing.
Also type comment must start with type name like `// Asset represents a release attachments`